ARTICLE.md

```diff
@@ -1,59 +1,144 @@
-# Introduction to property based testing
-Mankind has evolved - perhaps it's time for how we test software to evolve as well?
+---
+title: From 10 to 1,000,000 test cases in under an hour: A beginner's guide to property-based testing 
+description: This guide walks you through the what and why of property-based testing, with practical use cases and examples along the way. 
+author: Fredrik Fornwall
+authorLink: https://dev.to/fornwall
```

Comment:
So I've been thinking about this... For the website, we can have two authors - or if we can't yet, I can open a PR to make it so we can 😈 For the DEV article, one of us can post it and maybe give a little shout out to the other in the first line of the article? Not sure how we choose who posts it - could be based on followers, credibility, etc. 🤷‍♀ 

We can decide later, just some thoughts 😆 @fornwall

Reply:
Yea, awaiting https://github.com/thepracticaldev/dev.to/issues/1299 (which doesn't seem close to happening), a proposal is to:

- Write the shout out in the first line as you say ("This article was written together with XXX.") or similar.
- How about alternating, letting every other post be posted from one author (if the co-written article is roughly equally divided in work done, which might be a good thing to strive for)?

Reply:
We could do both? Alternate and give credit right away for the timebeing?

Thanks for flagging the issue, I'll be watching it now 👀

Reply:
Yes, I meant both!